learn/getting_started/quick_start.md

```diff
@@ -81,7 +81,15 @@ See our guide on [asynchronous updates](/learn/advanced/asynchronous_updates.md)
 
 ## Search
 
-Now that your documents have been ingested into MeiliSearch, you are able to search them.
+Before you start querying, we recommend that you check the status of your index. You can do that using:
+
+<CodeSamples id="get_update_1" />
```

Comment:
So, I'm not sure about this.

I believe that at this point in the quick start the user has made only one request triggering creation of an update object: the document addition request.

I also believe that the first update created for a MeiliSearch index will have the `updateId` of `0`. This code sample gives the value `1`, so I'm concerned that it might not work for users copy+pasting this command. Could you test it out yourself @maryamsulemani97 ?

Reply:
You're right, it should be `0`.